stable/presto/templates/coordinator-configmap.yaml

```diff
@@ -117,6 +117,18 @@ data:
     cp {{ .Values.server.properties.mapPath }}/*.config $PRESTO_CONF_DIR
     cp {{ .Values.server.properties.mapPath }}/*.json $PRESTO_CONF_DIR
     cp {{ .Values.server.properties.catalogMapPath }}/*.properties $PRESTO_CONF_DIR/catalog
+    cp {{ .Values.server.properties.thirdPartyMapPath }}/* ${PRESTO_HOME}/lib/third-party/
+
+    # make sure that oracle jdbc driver exists at the ap-node, otherwise remove oracle connector plugin from presto
+    if [[ -f ${PRESTO_THIRD_PARTY_LIBS}/{{ .Values.server.properties.thirdPartyLibs.oracleJdbcDriver }} ]]; then
+      # Create symbolic link to Oracle JDBC driver to enable Oracle connector
+      if [ -d ${PRESTO_HOME}/plugin/oracle ]; then
+        ln -s ${PRESTO_THIRD_PARTY_LIBS}/{{ .Values.server.properties.thirdPartyLibs.oracleJdbcDriver }} ${PRESTO_HOME}/plugin/oracle/{{ .Values.server.properties.thirdPartyLibs.oracleJdbcDriver }}
+      fi
+    else
+      # delete oracle connector plugin to make Presto server operational
+      rm -rf ${PRESTO_HOME}/plugin/oracle
+    fi
 
```

Comment:
what will happen if a client didn't have a driver in the beginning so plugin/oracle folder will get deleted and then he got a driver, put it the third lib folder and restart? will the plugin/oracle folder will get back?

Reply:
restarting presto service will re-apply helm-charts and re-create the pods. this mean that everything will be created fresh and all conditions will be re-evaluated.